kubeadm/flannel/flannel-host-gw.yml

```diff
@@ -179,9 +179,9 @@ spec:
         effect: NoSchedule
       containers:
       - name: kube-flannel
-        image: sigwindowstools/flannel:0.12.0
+        image: sigwindowstools/flannel:v0.13.0
```

Comment:
This added a `v` in the image. It would make it more consistent but would break the pattern of the older images

Reply:
I know. I added `v` because `quay.io/coreos/flannel` has it.

Reply:
Do you think it was bad idea?

Reply:
I think I've been burned but the missing `v` more than once but wonder about consistency, If we use end up going with the `nanoserver` suffix this wouldn't be to much of an issue.